allennlp/tests/tools/py2md_test.py

```diff
@@ -0,0 +1,46 @@
+from typing import Optional
+
+import pytest
+
+from allennlp.tools.py2md import py2md, Param, DocstringError
+from allennlp.common.testing import AllenNlpTestCase
+
+
+class TestPy2md(AllenNlpTestCase):
+    def test_basic_example(self, capsys):
+        py2md("allennlp.tests.fixtures.tools.py2md.basic_example")
+        captured = capsys.readouterr()
+
+        with open(self.FIXTURES_ROOT / "tools" / "py2md" / "basic_example_expected_output.md") as f:
+            expected = f.read()
+
+        assert captured.out == expected
+
+
+@pytest.mark.parametrize(
```

Comment:
Not really sure what you gain from this pattern, it just seems like it makes the test harder to read than just having a list of asserts... 🤷‍♂️

Reply:
Ha yea, I thought I would have a lot more cases. Actually I should probably add some more.